wikitongues/wikitongues/data_store/airtable/airtable_language_data_store.py

```diff
@@ -1,5 +1,6 @@
 from ..language_data_store import LanguageDataStore
 from ..error_response import ErrorResponse
+from wikitongues.wikitongues.data_store.airtable.offset_utility import OffsetUtility  # noqa: E501
```

Comment:
You should be able to use a relative import here:

```
from .offset_utility import OffsetUtility
```

Reply:
Thank you!